pkgs/misc/tmux-plugins/default.nix

```diff
@@ -957,6 +957,24 @@ in rec {
       maintainers = with maintainers; [ o0th ];
     };
   };
+  kanagawa = mkTmuxPlugin rec {
+    pluginName = "kanagawa";
+    version = "1.0.0";
```

Comment:
also, as there is no official release, the version should be `0-unstable-$(date of last commit)`